definitions/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "definitions",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "ts-node ./build.ts",
+    "test": "ts-node ./test-types.ts"
+  },
+  "dependencies": {
+    "tapable": "../"
```

Comment:
I think this uses the typings provided by `package.json: "typings": "./types.build.d.ts"`